MovieTime/MovieTime/Core/MTServiceHelper.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MTServiceHelper.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+protocol MTServiceHelperProtocol {
+    func buildURL(for baseURL: URL?, pathComponent: String, and parameters: MTParameters?...) -> URL?
+}
+
+struct MTServiceHelper: MTServiceHelperProtocol {
+
+    func buildURL(for baseURL: URL?, pathComponent: String, and parameters: MTParameters?...) -> URL? {
+        guard let url = baseURL?.appendingPathComponent(pathComponent) else { return nil }
+
+        var components = URLComponents(url: url, resolvingAgainstBaseURL: true)
+
+        if !parameters.isEmpty {
+            var queryItems: [URLQueryItem] = components?.queryItems ??  []
+            for params in parameters {
+                queryItems.append(contentsOf: buildQueryItems(parameters: params))
```

Comment:
I think you can replace the loop by `queryItems += parameters.map { buildQueryItems(parameters: $0) }`